.github/workflows/ci.yml

```diff
@@ -12,13 +12,17 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Lint
+    - name: Setup
       uses: actions/setup-node@v2
       with:
         node-version: '16.x'
         cache: 'yarn'
 
+    - name: Install Dependencies
+      run: yarn
+
     - name: Lint with ES Lint
-      run: |
-        yarn
-        yarn lint
+      run: yarn lint
+
+    - name: Build
```

Comment:
Won't there be any issues with Prisma and environment variables? I think I had these issues over at https://github.com/filiptronicek/CountDowner.

Reply:
What kind of issue?

Reply:
Well there is no database, so wouldn't it have a problem generating the client when it is not accessible?

Reply:
Ooo wait just updated the step to yarn build, if failed we can set env vars in gh secrets 🙇🏻

Reply:
Oops CI fails but it is unrelated to DB, its a problem with another breaking change in Next.js 😭

Reply:
Wonders, working now